src/test/java/BankAccTest.java

```diff
@@ -0,0 +1,53 @@
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+public class BankAccTest {
+
+        private BankAccountId bankAccountId;
+
+
+        @BeforeClass
+        public static void beforeClass() {
+            System.out.println("Before class");
+        }
+        @AfterClass
+        public static void afterClass() {
+            System.out.println("After cass");
+        }
+
+        @Before
+        public void intTest(){
+            bankAccountId = new BankAccountId();
+        }
+        @After
+        public void aftertest(){
+            System.out.println("for all");
+        }
+
+
+    @Test
+    public void credit(){
```

Comment:
name of the test method should be more clear, testCredit or verifyThatCreditWorksWith(some example of test data)